components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -211,60 +217,93 @@ function TeamUsage() {
                         )}
                         {billedUsage.length > 0 && !isLoading && (
                             <div className="flex flex-col w-full mb-8">
-                                <ItemsList className="mt-2 text-gray-500">
-                                    <Item header={false} className="grid grid-cols-5 bg-gray-100 mb-5">
-                                        <ItemField className="my-auto">
+                                <ItemsList className="mt-2 text-gray-400 dark:text-gray-500">
+                                    <Item header={false} className="grid grid-cols-12 gap-x-3 bg-gray-100 mb-5">
```

Comment:
**nitpick**: Minor spacing issue. We don't need so much negative space between the header and the list as this can make these two feel unrelated. See also [principles of grouping](https://en.wikipedia.org/wiki/Principles_of_grouping). 🤓

```suggestion
                                    <Item header={false} className="grid grid-cols-12 gap-x-3 bg-gray-100">
```

Reply:
Resolving as this has been fixed.